src/Components/PriceRange/PriceRange.tsx

```diff
@@ -75,6 +75,37 @@ export const PriceRange: FC<PriceRangeProps> = ({
 
   return (
     <>
+      {bars && shouldDisplayHistogram(bars) ? (
+        <Histogram
+          bars={bars}
+          selectedRange={[sliderRange[0], sliderRange[1]]}
+          data-testid="PriceFilterHistogram"
+        />
+      ) : null}
```

Comment:
Both versions are fine, but personally, I find this one easier to read:

```suggestion
      {!!bars && !!shouldDisplayHistogram(bars) && (
        <Histogram
          bars={bars}
          selectedRange={[sliderRange[0], sliderRange[1]]}
          data-testid="PriceFilterHistogram"
        /> }
```